virttest/qemu_devices/qcontainer.py

```diff
@@ -2243,6 +2243,7 @@ def chardev_define_by_params(self, chardev_id, params, file_name=None):
         _backend = 'socket' if 'socket' in backend else backend
         # Generate -chardev device
         chardev_param = Params({'backend': _backend, 'id': chardev_id})
+        file_name = '/dev/ttyS0' if backend == 'tty' else file_name
```

Comment:
`ttyS0` is only for x86 platform. @nanliu-r please confirm if this case need cover other platform, thanks!

Reply:
> `ttyS0` is only for x86 platform. @nanliu-r please confirm if this case need cover other platform, thanks!

yes, this case is "RHEL and x86 only"

Reply:
I just checked with ppc platform, it support tty too although we do not test it:
```
-chardev tty,id=id,path=path[,mux=on|off][,logfile=PATH][,logappend=on|off]
```